core/controllers/base_test.py

```diff
@@ -179,15 +179,6 @@ def setUp(self):
         # Update a config property that shows in all pages.
         self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
         self.login(self.ADMIN_EMAIL, is_super_admin=True)
-        response = self.testapp.get('/admin')
-        csrf_token = self.get_csrf_token_from_response(response)
-        self.post_json('/adminhandler', {
```

Comment:
Yes, I agree @seanlip. I was planning to add the test back. This was just so I could run tests on TravisCI. My internet was choppy last night. I wanted to leave a message stating that this is not ready to be merged.

Also I cannot run all the tests in my vagrant env. I get the following error:
```
INFO [karma]: Karma v0.12.16 server started at http://localhost:9876/
INFO [launcher]: Starting browser Chrome
ERROR [launcher]: Cannot start Chrome

INFO [launcher]: Trying to start Chrome again (1/2).
ERROR [launcher]: Cannot start Chrome

INFO [launcher]: Trying to start Chrome again (2/2).
ERROR [launcher]: Cannot start Chrome

ERROR [launcher]: Chrome failed 2 times (cannot start). Giving up.
```

Hence I could not run all the tests locally. I will see if there is a way to fix it.